core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -61,6 +61,14 @@ object RawlsApiService extends LazyLogging {
         )
         Sentry.captureException(rollback)
         complete(StatusCodes.InternalServerError -> ErrorReport(rollback))
+      case sql: SQLException =>
+        val sentryId = Sentry.captureException(sql)
+        logger.error(
+          s"Unhandled SQL exception with sentry id [$sentryId]: ${sql.getMessage} [${sql.getErrorCode} ${sql.getSQLState}] ${sql.getNextException}",
+          sql
+        )
+        val message = s"Internal server exception with sentry id: [${sentryId.toString}]"
```

Comment:
If anyone has better wording for this, I'd love to hear it.

Reply:
```suggestion
        val message = s"Internal server exception [sentryId=${sentryId.toString}]"
```

This would be a bit more consistent with other exceptions we throw during various workspace operations, but I think what you have works perfectly well too